packages/go_router_builder/example/lib/shell_route_with_keys_example.dart

```diff
@@ -0,0 +1,169 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Can you also add a test for this? just to make sure the code can actually compile

Reply:
I've added a simple test here: [Test shell_route_with_keys_example](https://github.com/flutter/packages/pull/3439/commits/28e593f917696da42a0e0169b247796a464f6b30)